config/default/forms/app/pregnancy_facility_visit_reminder.xml

```diff
@@ -102,8 +102,8 @@
           <patient_name tag="hidden"/>
           <patient_short_name tag="hidden"/>
           <patient_short_name_start tag="hidden"/>
-          <pregnancy_uuid_ctx tag="hiden"/>
-          <visit_date_for_task/>
+          <pregnancy_uuid_ctx/>
+          <visit_date_for_task tag="hidden"/>
```

Comment:
Why is the visit date now hidden? Is there an issue for that?

Reply:
I will revert this since we have a translation for it, and it is easier to hide in the future than bring it back for existing reports.

Reply:
I was correcting a misspelling.